tests/integration/routes/test_view_submitted_response_pdf.py

```diff
@@ -0,0 +1,66 @@
+from time import sleep
+
+from app import settings
+from tests.integration.routes.test_view_submitted_response import (
+    ViewSubmittedResponseBase,
+)
+
+
+class TestViewSubmissionResponsePDF(ViewSubmittedResponseBase):
+    def test_download_when_submitted_response_not_enabled(self):
+        # Given I launch and complete a questionnaire that does not have view-submitted-response enabled
+        self.launchSurvey("test_confirmation_email")
+        self.post()
+        self.post()
+
+        # When I try to download my answer
+        self.get("/submitted/download-pdf")
+
+        # Then a 404 page is returned
+        self.assertStatusNotFound()
+
+    def test_download_when_submitted_response_enabled_but_not_expired(self):
+        # Given I launch and complete a questionnaire that does not have view-submitted-response enabled
```

Comment:
this comment contradicts the test , is `view-submitted-response` enabled or disabled ?